src/file_manager/file_manager.py

```diff
@@ -0,0 +1,73 @@
+# file_manager.py
+# author: Josue Mendoza
+# date: 4-2-2015
+
+from xml.dom.minidom import parseString
+import os
+
+class File(object):
+	def __init__(self, file_path):
+		self.file_path = file_path
+
+	def read_content(self):
+		"""Uses native python implementations to read files,
+		it also handles IOError related exceptions.
+
+		It uses the file_path attribute (absolute file path) of the given
+		File instance
+
+		It returns a string containing the content read
+		"""
+		fo = open(self.file_path, 'r')
+
+		try:
+			file_content = fo.read()
```

Comment:
always provide an initial value for your return variables, if for some reason you don't reach this line, the return value will throw an exception for a non recognized variable.